packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -141,23 +143,38 @@ interface EpicHeaderProps {
     text: string;
     numArticles: number;
     tracking?: OphanTracking;
+    showAboveArticleCount: boolean;
 }
 
 const EpicHeader: React.FC<EpicHeaderProps> = ({
     text,
     numArticles,
     tracking,
+    showAboveArticleCount,
 }: EpicHeaderProps) => {
-    const elements = replaceArticleCount(text, numArticles, 'epic', tracking);
+    const elements = replaceArticleCount(
+        text,
+        numArticles,
+        'epic',
+        tracking,
+        !showAboveArticleCount,
+    );
```

Comment:
Just trying to work through the logic here - so when we `showAboveArticleCount`, we just replace with text. This works nicely for this test. It might also affect tests in the tool that have been configured both to show the above AC & have ac in the body copy - do we need to check that's okay?